sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeriesIdOfT1.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is composed of one primitive value that uniquely identifies a Time Series instance.
+    /// </summary>
+    /// <remarks>
+    /// A single Time Series Id can be composite if multiple properties are specified as Time Series Id at environment creation time.
```

Comment:
This is actually good, I would add the same in extended classes as customers would use it directly.

& Do you think this should be para instead?

Reply:
The other extended classes have been removed. 

Regarding the para vs remarks, I looked up the documentation for this. A <para> should exist inside a <remarks>. I will wrap the text in a para tag to represent a paragraph.